pages/linux/dunstify.md

```diff
@@ -0,0 +1,16 @@
+# dunstify
+
+> A notification tool that is an extension of notify-send, but has more features based around dunst.
+> Works with all options that work for notify-send.
+
+- Show a notification with title "Title" and message "Message":
+
+`dunstify {{"Title"}} {{"Message"}}`
+
+- Show a notification with specified urgency (options: "low", "normal", "high"):
```

Comment:
Perhaps this will read better?
```suggestion
- Show a notification with specified urgency:
```
Note the next suggestion too, which goes with this one.